### ISSUE TITLE
Tutorial about EnTK/RCT at SEA: Apr 2019

### ISSUE THREAD

**Issue by mturilli** [radical-collaboration/hpc-workflows#70]
Depends on publication from PSU.

**Comment by mturilli** [comment]
This is going to happen

**Comment by mturilli** [comment]
To be discussed in the next meeting as a follow up with the organizer.

**Comment by Weiming-Hu** [comment]
[SEA 2019](https://sea.ucar.edu/conference/2019)

**Comment by Weiming-Hu** [comment]
General question: do all users need to have their thread limit changed if they want to run it on Cheyenne?

**Comment by vivek-bala** [comment]
Based on the last email exchanges with the sys admins, yes. 

We can maybe ask them to revisit increasing the limits, at least for a subset of
users temporarily -- i.e. any SEA users if that's where your question was headed. They 
did say they might consider it after monitoring our (as in Weiming's and Vivek's) 
utilization after the limit increase.

**Comment by mturilli** [comment]
Starting and sharing a document with the current information and ideas about the tutorial.

**Comment by mturilli** [comment]
Parked until confirmed due to the shutdown. Send email and see whether we can get more information.

**Comment by mturilli** [comment]
- Presentation Analog/EnTK on Tuesday 8
- 1/2 day tutorial on Thursday 10
- Create/share tutorial googledoc document.

**Comment by Weiming-Hu** [comment]
https://sea.ucar.edu/conference/2019/program

**Comment by Weiming-Hu** [comment]
Proposed narrative (on-going) [here](https://drive.google.com/drive/folders/1x4Y7yjl11ICI4cRdPkT0jPAS49ruBo2A?usp=sharing)

**Comment by mturilli** [comment]
Depends on #86
